cmd/arrai/eval.go

```diff
@@ -19,7 +19,7 @@ var evalCommand = &cli.Command{
 }
 
 func evalImpl(source string, w io.Writer) error {
-	return evalExpr(syntax.NoPath, source, w)
+	return evalExpr(".", source, w)
```

Comment:
Is this change necessary? The original intent was to disallow imports when there is no source file.

Reply:
Yes. Actually `"."` is the original code. I thought `syntax.NoPath` would be better so I changed it to NoPath. Then I found out `arrai e "//./examples/simple/simple"` cannot find the right file path with `NoPath`.

Reply:
Sorry, I meant to retract this yesterday when I realised it makes perfect sense, but forgot. All good.